.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
   - DJANGO="Django>=1.11,<1.12"
   - DJANGO="Django>=2.0,<2.1"
   - DJANGO="Django>=2.1,<2.2"
+  - DJANGO="Django>=2.2,<2.2"
```

Comment:
`<2.3`?

Reply:
Ah thank you. Actually just tried running the tests locally and they failed 😞